data/news/alameda.json

```diff
@@ -30,7 +30,7 @@
     {
       "id": "https://covid-19.acgov.org/covid19-assets/docs/press/joint-statement-2020.09.28.pdf",
       "url": "https://covid-19.acgov.org/covid19-assets/docs/press/joint-statement-2020.09.28.pdf",
-      "title": "Guidance on How to Celebrate Halloween and D\u00eda de Los Muertos Safely",
+      "title": "Guidance on How to Celebrate Halloween and D",
```

Comment:
So it looks like we are hitting more crazy/weird formatting on the Alameda press page. In this case, there is a link in the title, but it doesn’t encompass the whole title — it’s just a single letter!

<img width="988" alt="Screen Shot 2020-10-09 at 9 31 01 AM" src="https://user-images.githubusercontent.com/74178/95608585-2ddbca00-0a12-11eb-9167-7a902b2acec9.png">

That’s clearly wrong, but I think we could still handle it better.

I think we can hold off on merging this PR for now, since there are no substantive additions (yet!) and only removals of old news. Those aren’t critical news updates we need to show on the site.

Reply:
Track the fix for this in sfbrigade/data-covid19-sfbayarea#140